Gopkg.toml

```diff
@@ -73,3 +73,8 @@ required = [
   [[prune.project]]
     name = "k8s.io/gengo"
     non-go = false
+
+[[constraint]]
```

Comment:
For neatness, I'd suggest adding this above the `[prune]` section. Additionally, if there are a bunch of packages in it that we don't make use of, you could add a `[[prune.project]]` section under `[prune]` that will limit the number of useless files that we check in (I wish I'd known about this before checking in the vendor dir initially...for the already committed stuff it wouldn't help that much to remove them now):

```toml
  [[prune.project]]
    name = "github.com/enmasseproject/enmasse"
    unused-packages = true
```